admin/database/database.go

```diff
@@ -151,6 +152,17 @@ type Project struct {
 	ProductionDeploymentID *string   `db:"production_deployment_id"`
 	CreatedOn              time.Time `db:"created_on"`
 	UpdatedOn              time.Time `db:"updated_on"`
+	Envs                   Envs      `db:"env"`
+}
+
+type Envs map[string]string
+
+func (e *Envs) Scan(value interface{}) error {
+	b, ok := value.([]byte)
+	if !ok {
+		return errors.New("failed type assertion to []byte")
+	}
+	return json.Unmarshal(b, &e)
 }
```

Comment:
I'm wondering if it would be better to make this a generic `type JSON any` type, and then add a `GetEnv()` accessor on `Project` that casts it to a `map[string]string`. But that's also ugly in it's own way.

I would probably still keep the interface clean and add the extra lines of code in the implementation, but I'm fine with any approach.